x-pack/plugins/canvas/public/components/asset_picker/asset_picker.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { EuiFlexGrid, EuiFlexItem, EuiLink, EuiImage, EuiIcon } from '@elastic/eui';
+
+export class AssetPicker extends PureComponent {
+  static propTypes = {
+    assets: PropTypes.array,
+    selected: PropTypes.string,
+    onChange: PropTypes.func,
+  };
+
+  componentDidMount() {
+    const selectedAsset = document.getElementById('canvasAssetPicker__selectedAsset');
+    if (selectedAsset) selectedAsset.scrollIntoView();
```

Comment:
I don't think this is quite right, MDN says the following:

> The `Element.scrollIntoView()` method scrolls the element on which it's called into the visible area of the browser window.

I think this means that it'll scroll the entire document body to get the element into view, not just its scrollable container. I may be wrong, but with the currently broken page preview layout in master, I end up with this view once I pick an asset:

![screenshot 2018-12-03 12 38 42](https://user-images.githubusercontent.com/404731/49397308-6284e900-f6f8-11e8-8513-05beeba8fadd.png)

Normally this would still look right, since the size of the application is "fixed", but isolating the scroll action to the asset container would be safer.

Reply:
That said, it's probably fine for this PR, since it only behaves weird because the layout is currently broken. It can be changed later if needed.

Reply:
I've got a fix for the page preview bug #26573, so it shouldn't be an issue once both are merged. 

I would've prefered to use `Element.scrollTo()` which isolate the scroll to the asset picker, but it's not supported in IE11 whereas there is basic support for `Element.scrollIntoView()` in IE11.

Reply:
> not supported in IE11

😢 

> We can explore other ways of preserving the scroll if it presents to be a problem later on.

👍